### ISSUE TITLE
Policy Tests (Collection Runner) need to have special BC rules

### ISSUE THREAD

**Issue by milechukwu** [AustenR/PMTestingSuites#6]
I just ran GAL through the RobotBaby and found something in the results for minimum stays.  The resorts in BC have their own unique set of rules for certain things:

- **sites**: no minimum stays ever
- **rentals**: the only minimum-stay rule is a 2-night minimum any day of the week during a peak season.  There is no special minimum over long weekends; just that same 2-night minimum during peak.  No minimum ever during shoulder or off-peak.

I have two requests:
1. short-term: can the RobotBaby recognize certain BC resorts and use the above unique rules for policy testing?
1. long-term: let's consider a new feature of RobotBaby that would allow policies per resort to be configurable somehow.  Not build it just yet, just discuss what would be involved.

**Comment by AustenR** [comment]
The Minimum Stay policy test has been changed to include BC resort specifics. The created datasheets no longer have test cases that are irrelevant. For example, no sites in the shoulder or the off-peak will be tested.

BC: tests only rentals for 2 night minimum all year

Non-BC: (Only Peak)
- Rentals 2 night min (3 nights for long weekends)
- Sites 2 night min (3 nights for long weekends)

This test executes by attempting to book a reservation for one day less than the minimum.

eg.
- successful tests appear in the summary logs: "Staying 2 Nights when a Minimum of 3 is Required."
- failed tests appear in the error logs: "1 night stay is allowed"

attached is an error and summary logs for a CAL min stay test.

@milechukwu Let me know if this is what you were expecting, I was trying to find a BS resort to test this with (I believe none have their 2019 rates available for testing currently), I did use OCS to test though OCS does not have any rentals, therefore there was no tests run.

[Night Min Stay-Errors (10-30).xlsx](https://github.com/AustenR/PMTestingSuites/files/2530650/Night.Min.Stay-Errors.10-30.xlsx)
[Night Min Stay-Summary (10-30).xlsx](https://github.com/AustenR/PMTestingSuites/files/2530651/Night.Min.Stay-Summary.10-30.xlsx)
